pyls/plugins/__init__.py

```diff
@@ -2,11 +2,11 @@
 from . import (
     completion, definition, format,
     hover, pyflakes_lint, pycodestyle_lint,
-    references, symbols
+    references, symbols, signature
```

Comment:
Fixed, that was so close already :)
About building PRs from forks, is it a fixable problem? Circle's documentation mentioned the risk of leaking secrets?